upgrading/upgrade-helm.adoc

```diff
@@ -7,7 +7,11 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-You can upgrade to the latest version of {product-title} from a supported older version. For upgrading to {product-title-short} 4.0, you must be using the latest patch release of {product-title-short} 3.74. If you are using an older version, you must first upgrade to {product-title-short} 3.74.
+When upgrading, follow this guidance:
+
+* If Central is earlier than 3.74, you must upgrade to 3.74 before upgrading to a 4.x release. For upgrading Central from version 3.74 to 4.x, see the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74].
+* For Operator- and Helm-based Central deployments, you can upgrade from release 3.74 to any latest patch of {product-title-short} version 4.0 through 4.4. However, for full functionality, upgrade to the latest version.
```

Comment:
1. We don't need to mention Operator here because these are instructions for Helm.
2. I suggest to make it more explicit that 3.74 -> 4.0..4.4 is ok but 3.74 -> 4.5+ is not ok.
3. I like your point about upgrading to the latest but I suggest to reword it slightly so that the proposed action is clear.

```suggestion
* When upgrading Helm-based Central deployments from release 3.74, choose any latest patch of {product-title-short} version 4.0 through 4.4 but not 4.5 or later. After that, upgrade Central to the latest version.
```

Also, a link doc from 4.4's documentation would be useful.

Reply:
> 2\. I suggest to make it more explicit that 3.74 -> 4.0..4.4 is ok but 3.74 -> 4.5+ is not ok.

This PR isn't for 4.5, this is for 4.4 and earlier. I'll make a separate PR for 4.5.

Reply:
Rewrote because the intent was to tell them to upgrade to 4.4.x (the latest version, as of this publication - 4.4 version).

Reply:
And I can add a link to the 4.4 docs from the 4.5 docs if you think that's necessary. (per your comment above)